core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -350,7 +356,7 @@ public String toString() {
           .setDescription("The file path of the metrics system configuration file. By default "
               + "it is `metrics.properties` in the `conf` directory.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
-          .setScope(Scope.SERVER)
+          .setScope(Scope.ALL)
```

Comment:
Does this really apply to `ALL`? For clients, this defines a path to a metrics properties file which might not necessarily exist if the client is remote to Alluxio.

Reply:
Client will also check this file and start sink metrics if the file exists.